starter-code/styles/modules.css

```diff
@@ -10,13 +10,15 @@
 }
 
 .icon-menu {
-  /* TODO: Add styles for the icon-menu class (whether it be the
-      hamburger font icon or an image of your choosing). It should be
-      bigger, and off to the right. */
+      float: right;
+      width: 100px;
```

Comment:
Great solution here! Another possibility could be to use the font-size property, since icon fonts can be treated like text.